scio-test/src/test/scala/com/spotify/scio/values/SCollectionPropTest.scala

```diff
@@ -0,0 +1,37 @@
+package com.spotify.scio.values
+
+import com.fortysevendeg.scalacheck.datetime.joda.ArbitraryJoda._
+import com.spotify.scio.testing.PipelineSpec
+import org.joda.time._
+import org.scalacheck.{Arbitrary, Prop}
+import org.scalatest.prop.Checkers
+
+import scala.reflect.ClassTag
+
+class SCollectionPropTest extends PipelineSpec with Checkers {
```

Comment:
What's the need for pulling `ScioContext`/pipeline for this? This should be part of tests in `com.spotify.scio.coders.*`, and no need to pull `ScioContext`, instead use `CoderTestUtils`'s `testRoundTrip`.